ArduinoCode/Driver-IO/RTDS/RTDS/RTDS.ino

```diff
@@ -0,0 +1,68 @@
+//By Joshua Cheng
+//Oct 12, 2020
+
+/* detects shutdown voltage, if voltage is OK (12v scaled down) allow button (turn on button LED) 
+ *  to turn speaker on for three seconds 
+ */
+
+bool shutdownVoltage = false;
+const int SHUTDOWN_DETECTION_PIN = 3; //pin for detecting shutdown system voltage
+
+const int BUTTON = 2; 
+const int BUTTON_LED = 11;
+const int SPEAKER = 10; //must be pwm pin
+long speakerCooldown = 0;
+long buttonCooldown = 0;
+bool state = false;
+
+
+
```

Comment:
Add JavaDoc-style commenting to all functions. see [this article](https://developer.lsst.io/cpp/api-docs.html) for some guidelines.